tests/SprykerEcoTest/Zed/ArvatoRss/Business/Api/Converter/RiskCheckResponseConverterTest.php

```diff
@@ -55,4 +58,59 @@ protected function createExpectedResult($response)
             ->setActionCode($response->Decision->ActionCode)
             ->setResultText($response->Decision->ResultText);
     }
+
+    /**
+     * @param RiskCheckResponseConverter $converter
+     */
+    protected function testDeliveryPresents(RiskCheckResponseConverter $converter)
+    {
+        $result = $converter->convert($this->createResponseWithDelivery());
+
+        $this->assertInstanceOf(
+            ArvatoRssAddressValidationResponseTransfer::class,
+            $result->getDeliveryAddressValidation()
+        );
+
+        $result = $converter->convert($this->createResponseWithBilling());
+        $this->assertEmpty($result->getDeliveryAddressValidation());
+    }
+
+    /**
+     * @return stdClass
+     */
+    protected function createResponseWithDelivery()
+    {
+        $response = $this->createResponseWithBilling();
```

Comment:
Please create function with code
```
 +        $response = $this->createResponse();
 +        $response->Details = new stdClass();
 +        $response->Details->BillingCustomerResult = new stdClass();
 +        $response->Details->BillingCustomerResult->ServiceResults = new stdClass();```
and reuse it, with such 'override' calls you may get unexpected results for some internal properties.

Reply:
If you need to add different data, please create functions like 'addBillingTestData', 'addDeliveryTestData', etc and call them one after another when required.